src/Rokt-Kit.js

```diff
@@ -238,6 +273,39 @@ var constructor = function () {
         }
         return {};
     }
+
+    function fetchOptimizelyFeatureExpAllocations(userId, attributes) {
+        var forwarders = window.mParticle
+            ._getActiveForwarders()
+            .filter(function (forwarder) {
+                return forwarder.name === 'Optimizely';
+            });
+
+        if (forwarders.length > 0 && window.optimizelyClient) {
+            var user = window.optimizelyClient.createUserContext(
+                userId,
+                attributes
+            );
+            var config = window.optimizelyClient.getOptimizelyConfig();
```

Comment:
same comment regarding this one. check for `window.optimizelyClient.hasOwnProperty('getOptimizelyConfig')`